src/services/legal-services.ts

```diff
@@ -167,6 +168,19 @@ export default class LegalServices {
       })
   }
 
+  /**
+   * Creates (posts) a draft (temporary) business record.
+   * Must be logged in to use this.
+   * Throws an exception on error.
+   */
+  static async createBusiness (businessRequest: any): Promise<any> {
+    const axios = addAxiosInterceptors(Axios.create())
```

Comment:
Why do you need this?

Reply:
You're right. I just haven't focused working on creating the draft yet. I'll have an upcoming PR that'll handle that.

Reply:
Fixed, thanks.